settings

```diff
@@ -1,24 +1,34 @@
-VERSION="1.23"
+FOREMAN_VERSION="1.23"
```

Comment:
I've never felt very good about hardcoding this in the file. Wondering if this should also be read similar to `$PROJECT`.